tox.ini

```diff
@@ -58,9 +59,9 @@ description = Run pyflakes over the pydoctor code
 deps: pyflakes
 
 commands =
-    ; Custom pyflakes run to exlcude test files.
+    ; Custom pyflakes run to exlcude some files.
```

Comment:
why extend the extensions?
If for some reason non-test files need special handling for pyflakes, I think that the comment  should document that.

I think that pyflakes is the minium check that you should run to help keep the code clean.

So if possible, I would like not to skip pyflakes checks.

Reply:
This is only for the demo site, so I'de say it's ok. But I can add some `#noqa` comments instead, it will probably work ? 
Oh no, pyflakes does not support those I think...